src/components/modals/HelpModal.js

```diff
@@ -0,0 +1,111 @@
+import { Component } from 'react';
+import { div, form, input, label, textarea, hh } from 'react-hyperscript-helpers';
+import { BaseModal } from '../BaseModal';
+import { Alert } from '../Alert';
+import { Help } from '../../libs/ajax';
+import { Storage } from "../../libs/storage";
+
+export const HelpModal = hh(class HelpModal extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      subject: '',
+      description: '',
+    };
+    this.closeHandler = this.closeHandler.bind(this);
+    this.afterOpenHandler = this.afterOpenHandler.bind(this);
+    this.OKHandler = this.OKHandler.bind(this);
+    this.subjectChangeHandler = this.subjectChangeHandler.bind(this);
+    this.descriptionChangeHandler = this.descriptionChangeHandler.bind(this);
+  }
+
+  async OKHandler() {
+    const report = {};
+    report.description = this.state.description;
+    report.subject = this.state.subject;
+    report.userId = Storage.getCurrentUser().dacUserId;
+    await Help.createHelpMeReport(report);
+    await this.setState(prev => {
+      prev.subject = '';
+      prev.description = '';
+      return prev;
+    })
+    this.props.onOKRequest('help');;
+  }
+
+  closeHandler() {
+    // this is the method to handle Cancel click
+    // could do some cleaning here 
+    // or delegate it to the parent
+    // we need to use it to close the
+    // DO SOMETHING HERE ...
+
+    // and call parent's close handler
+    this.props.onCloseRequest('help');
+  }
+
+  afterOpenHandler() {
+    // DO SOMETHING HERE ...
```

Comment:
And here?

Reply:
unused library callback . We'll remove it.